src/bankid.ts

```diff
@@ -36,7 +38,6 @@ interface AuthOptionalRequirements {
 
 export interface SignRequest extends AuthRequest {
   userVisibleData: string;
-  userVisibleDataFormat?: "simpleMarkdownV1";
```

Comment:
Since we're changing this - we might as well move `userNonVisibleData` to `AuthRequest` as well.